### PR TITLE
Add custom line endings.

### PR DESCRIPTION
**Pull request by dan-jensen**
Added a way to apply custom line endings when calling "generate" on a document.  This is most useful for applying Windows \r\n endings.  Also added tests for this functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,6 +208,14 @@ end
 ```
 
 
+## Custom line endings
+
+The default line ending is '\n' (Linux/OSX standard). A different line ending can be applied by passing it as the 2nd argument to the Document's generate method.  For example, if you wanted to generate a document without debug_mode and apply Windows standard line endings, you would do this:
+
+```ruby
+PeopleDocument.new.generate(false, '\r\n')
```
Comment (Chetane) [inline README.md]: Technically, this should be `"\r\n"` (with double quotes).

File: lib/fixy/document.rb [hunk 2]
```diff
@@ -9,8 +9,9 @@ def generate_to_file(path, debug = false)
       end
     end
 
-    def generate(debug = false)
+    def generate(debug = false, line_ending = "\n")
```
Comment (Chetane) [inline lib/fixy/document.rb]: For consistency, I would also add this option to the `generate_to_file` method on line 6.

File: spec/fixy/document_spec.rb [hunk 3]
```diff
@@ -50,5 +50,10 @@ def build
       ParsedPeopleDocument.new.generate.should eq "Arcturus  Mengsk    \nSarah     Kerrigan  \nJim       Raynor    \n"
       ParsedPeopleDocument.new.generate(true).should eq File.read('spec/fixtures/debug_document.txt')
     end
+
+    it 'should apply custom line endings if given' do
+      ParsedPeopleDocument.new.generate(false, "\r\n").should eq "Arcturus  Mengsk    \r\nSarah     Kerrigan  \r\nJim       Raynor    \r\n"
+      ParsedPeopleDocument.new.generate(true, "\r\n").should eq File.read('spec/fixtures/debug_document_custom_line_endings.txt')
```
Comment (Chetane) [inline spec/fixy/document_spec.rb]: nitpick: maybe the file should have an `html` extension.

**Comment by Chetane** [comment]
@dan-jensen Thanks for submitting this PR, and apologies for not seeing it earlier. It looks good, just a few small suggestions to consider.

**Comment by dan-jensen** [comment]
@Chetane And I just saw your suggestions after you mentioned me when you merged samkim's PR.  (I'm not sure why notifications weren't working for either of us.)  It looks like samkin's PR does a nice job with line endings, so I'm closing my PR.  Thanks for your good work.

**Pull request closed without merging**